includes/admin/class-wc-stripe-onboarding-controller.php

```diff
@@ -42,6 +42,13 @@ public function admin_scripts() {
 			$script_asset['version'],
 			true
 		);
+		wp_localize_script(
+			'wc_stripe_onboarding_wizard',
+			'wc_stripe_settings_params',
```

Comment:
Haha I named this so that it would be the same - but now that you're pointing it out, maybe it's best if they're named differently :D 
Renamed in 7b6dd53a 👍